.github/workflows/publish.yaml

```diff
@@ -0,0 +1,27 @@
+name: Publish
+on: [push]
+
+jobs:
+  publish:
+    name: Publish
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+           node-version: 16.x
+      - run: yarn install
+      - run: yarn npm:prepublish
+      - name: Set the package name
+        run: npm pkg set --workspace packages/server name=@deepnote/sql-language-server
```

Comment:
**suggestion:** I would maybe add a line that this does the renaming of original package.json name field :) it's not that super obvious but maybe not important as well...

Reply:
Sure. I can add comments (and was planning to). I assume by "line" you mean "comment", correct?